c++/src/capnp/compiler/module-loader.c++

```diff
@@ -207,6 +207,18 @@ public:
     return loader.getErrorReporter().hadErrors();
   }
 
+  void reportResolution(uint32_t startByte, uint32_t endByte, uint64_t typeId) override{
+    Resolution resolution;
+    resolution.startByte = startByte;
+    resolution.endByte = endByte;
+    resolution.target = Resolution::Type{typeId};
+    resolutions.add(kj::mv(resolution));
```

Comment:
```suggestion
    resolutions.add(Resolution {
      .startByte = startByte,
      .endByte = endByte,
      .target = Resolution::Type { typeId },
    });
```

Reply:
Thank you for your suggestion.
I will change reportResolution can take `Resolution` as a parameter , so I will refer to it in the implementation on the caller side.